content/plotly/concepts/graph-objects/terms/surface/surface.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: 'Surface'
+Description: 'Creates a 3d plot using the Surface class of the graph_object module in Plotly'

```

Comment:
```suggestion
Description: 'Creates a 3d plot using the `Surface` class of the `graph_objects` module in Plotly'

```
`Surface` and `graph_objects` must be enclosed in backticks(`.....`)

Reply:
@SaviDahegaonkar, thank you!